src/engraving/rendering/dev/beamtremololayout.cpp

```diff
@@ -1066,11 +1084,19 @@ double BeamTremoloLayout::chordBeamAnchorX(const ChordRest* cr, ChordBeamAnchorT
 
 double BeamTremoloLayout::chordBeamAnchorY(const ChordRest* cr) const
 {
+    const Chord* chord = toChord(cr);
+
+    if ((cr->staffType() && cr->staffType()->xmlName() == "stdJianpu")
+        && (cr->staffType() && cr->staffType()->lines() == 0)) {
+        Note* note = chord->downNote();
+        PointF position = note->pagePos();
+        return position.y() - 0.000001;
```

Comment:
Why `- 0.000001`? That looks like a hack. What is the problem that needs to be solved with this? Hopefully we can find a better solution.